modules/perception/camera/app/cipv_camera.h

```diff
@@ -82,6 +82,12 @@ class Cipv {
   static float VehicleDynamics(const uint32_t tick, const float yaw_rate,
                                const float velocity, const float time_unit,
                                float *x, float *y);
+  static float VehicleDynamics(const uint32_t tick, const float yaw_rate,
+                            const float velocity, const float time_unit,
```

Comment:
please adjust the tabs

Reply:
Thank you for the review! I fixed the indentation.